### PR TITLE
More features

### PR DESCRIPTION
**Pull request by SuperQ**
Add flag for target heap size.

Add a progress bar
* Shows progress.
* Shows ETA.
* Update README to send stderr log to file.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -14,19 +14,21 @@ import (
...
 	"github.com/prometheus/tsdb/labels"
+	"gopkg.in/cheggaaa/pb.v1"
 )
 
 func main() {
 	v1Dir := flag.String("v1-dir", "./data-v1", "Path to the v1 storage directory.")
 	v2Dir := flag.String("v2-dir", "./data-v2", "Path to the v2 storage directory.")
 	lookback := flag.Duration("lookback", 15*24*time.Hour, "How far back to start when exporting old data.")
 	step := flag.Duration("step", 15*time.Minute, "How much data to load at once.")
+	v1HeapSize := flag.Uint64("storage.local.target-heap-size", 2000000000, "How much memory to use for v1 storage in bytes")
```
Comment (juliusv) [inline main.go]: In this context, the old flag name is a bit misplaced... how about `-v1-target-heap-size`?

If we get several more flags, we can think about more namespacing again...

Comment (SuperQ) [reply main.go]: Fine with me, I was mostly thinking copy-n-paste from Prometheus server flags.

File: main.go [hunk 2]
```diff
@@ -14,19 +14,21 @@ import (
 	"github.com/prometheus/prometheus/storage/metric"
 	"github.com/prometheus/tsdb"
 	"github.com/prometheus/tsdb/labels"
+	"gopkg.in/cheggaaa/pb.v1"
```
Comment (juliusv) [inline main.go:17]: Run `dep ensure` (from https://github.com/golang/dep)

Comment (SuperQ) [reply main.go:17]: Done

**Comment by juliusv** [comment]
:+1: Thanks!

**Pull request merged**